File: Recursion/fibonacci.cpp
#include <iostream>

using namespace std;

int fibonacci(int n){
    return n<=1 ? n : fibonacci(n-1) + fibonacci(n-2);
}

int main(){
    int n;
    cout<<"Enter a number: ";
    cin>>n;
    cout<<"Fibonacci of the "<<n<<"th number is "<<fibonacci(n)<<endl;

    return 0;
}

File: Recursion/fibonacci.java
import java.util.*;

public class fibonacci{

    public int fibonacci(int n){
        return n<=1? n : fibonacci(n-1) + fibonacci(n-2); // where we are getting the n-th fibonacci number in the series
    }

    public static void main(String[] args) {
                
        fibonacci f = new fibonacci();
        System.out.println(f.fibonacci(4));
}
}